optaplanner-examples/src/main/resources/org/optaplanner/examples/scrabble/solver/scrabbleSolverConfig.xml

```diff
@@ -6,10 +6,15 @@
   <solutionClass>org.optaplanner.examples.scrabble.domain.ScrabbleSolution</solutionClass>
   <entityClass>org.optaplanner.examples.scrabble.domain.ScrabbleWordAssignment</entityClass>
   <entityClass>org.optaplanner.examples.scrabble.domain.ScrabbleCell</entityClass>
+  <environmentMode>FULL_ASSERT</environmentMode>
```

Comment:
This needs to be commented out before merging.

Reply:
Also, we usually keep it at the top of the file. (Check other examples for reference.)